src/components/FileTreeNode/index.tsx

```diff
@@ -86,11 +86,15 @@ const getIconForType = (type: string | null) => {
       return 'times-circle';
     case 'warning':
       return 'exclamation-triangle';
+    case 'known-library':
+      return 'check-circle';
     default:
       return 'info-circle';
   }
 };
 
+export const LINTER_MESSAGE_KNOWN_LIBRARY = 'JS Library Detected';
```

Comment:
Let's use the code instead. Here's a view of the API response:

<img width="370" alt="Screenshot 2019-03-19 16 28 29" src="https://user-images.githubusercontent.com/55398/54643147-32bddf00-4a64-11e9-8e2f-b30b842a7906.png">

More info: https://github.com/mozilla/addons-linter/blob/master/docs/rules.md

Reply:
oh, I could not find this constant. I found it in the linter but did not find it in the JSON file. I believe it is not stored in redux yet?